reflex/components/component.py

```diff
@@ -471,6 +471,11 @@ def get_event_triggers(self) -> Dict[str, Any]:
             EventTriggers.ON_MOUNT: lambda: [],
             EventTriggers.ON_UNMOUNT: lambda: [],
         }
+        # Look at the fields for EventHandler types.
+        for field in self.get_fields().values():
```

Comment:
A bit confused what this portion is doing, can we add more in the comments?

Reply:
how about using the same terms "component specific triggers"